src/jit/flowgraph.cpp

```diff
@@ -2415,25 +2415,51 @@ void Compiler::fgDfsInvPostOrderHelper(BasicBlock* block, BlockSet& visited, uns
     // Assume we haven't visited this node yet (callers ensure this).
     assert(!BlockSetOps::IsMember(this, visited, block->bbNum));
 
+    ArrayStack<DfsBlockEntry> stack(this);
+
+    DfsBlockEntry firstEntry(State_Pre, block);
```

Comment:
This was test-only because I haven't really cleaned up any of the CC conventions yet before I would submit for a CR.

But anyway this seems like preference to me. I personally like explicitly named variables as it can make the code more readable than huge single line expressions, and secondly in the debugger it makes it easier to examine the state of variables. Is there any consensus out there?